<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatchCreationDialog</class>
 <widget class="QDialog" name="MatchCreationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Match creation dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>681</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Participating Teams</string>
   </property>
   <widget class="QTableView" name="teams_tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>681</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Tournament selection</string>
   </property>
   <widget class="QComboBox" name="tournament_comboBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="choose_team1_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choose Team1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="choose_team2_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choose Team2</string>
   </property>
  </widget>
  <widget class="QLabel" name="team1_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Team1</string>
   </property>
  </widget>
  <widget class="QLabel" name="team2_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Team2</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>681</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Studios</string>
   </property>
   <widget class="QTableView" name="studios_tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>681</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>490</y>
     <width>681</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Commentators</string>
   </property>
   <widget class="QTableView" name="commentatros_tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>681</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="choose_studio_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choose Studio</string>
   </property>
  </widget>
  <widget class="QLabel" name="studio_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Studio</string>
   </property>
  </widget>
  <widget class="QLabel" name="commentator_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Commentator</string>
   </property>
  </widget>
  <widget class="QPushButton" name="choose_commentator_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choose Commentator</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>660</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>710</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MatchCreationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MatchCreationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
